_posts/2019-02-20-lecture-notes-template.md

```diff
@@ -0,0 +1,361 @@
+
+Using this information, we can compute the measurement update (update step) as:
+<d-math block>
+\begin{aligned}
+\hat X_{t+1 \vert t+1} &= \hat X_{t+1 \vert t} + K_{t+1} (Y_{t+1} - C \hat X_{t+1 \vert t}) \\
+P_{t+1 \vert t+1} &= P_{t+1 \vert t} - KCP_{t+1 \vert t}\\
+\end{aligned}
+</d-math>
+Here $K_{t+1} = P_{t+1 \vert t} C^T (C P_{t+1 \vert t} C^T + R)^{-1}$ and is known as the Kalman gain matrix. An interesting point to note is that $K_t$ is independent of $Y_t$ (i.e. the data), so it can be precomputed to improve efficiency.
+
+### Example of Kalman Filtering in 1-D
+Suppose we have some noisy observations of a particle performing a random walk in 1-D. Assume our states and observations are defined as follows: \\
+$ X_{t \vert t-1} = X_{t-1} + W $ where $W \sim \mathcal{N}(0,\sigma_x)$ \\
+$ Z_{t} = X_{t} + V $ where $V \sim \mathcal{N}(0,\sigma_z)$ \\
+We can compute KF updates for this model as follows:\\
+$P_{t+1 \vert t} = AP_{t \vert t}A + GQG^T = \sigma_t + \sigma_z$\\
+$\hat X_{t+1 \vert t} = A\hat X_{t|t} = \hat X_{t|t}$\\
+$K_{t+1} = P_{t+1 \vert t} C^T (C P_{t+1 \vert t} C^T + R)^{-1} = (\sigma_t + \sigma_x)(\sigma_t + \sigma_x + \sigma_z)$\\
+$\hat X_{t+1 \vert t+1} &= \hat X_{t+1 \vert t} + K_{t+1} (Z_{t+1} - C \hat X_{t+1 \vert t}) = \frac{(\sigma_t + \sigma_x)Z_t + \sigma_z \hat X_{t|t}}{\sigma_t + \sigma_x + \sigma_z}$
+$P_{t+1 \vert t+1} = \frac{(\sigma_t + \sigma_x) \sigma_z}{\sigma_t + \sigma_x + \sigma_z}$
+
+### Understanding the intuition behind Kalman Filtering
+In the KF update equation for the mean, $\hat X_{t+1 \vert t+1} &= \hat X_{t+1 \vert t} + K_{t+1} (Z_{t+1} - C \hat X_{t+1 \vert t})$, the term $(Z_{t+1} - C \hat X_{t+1 \vert t})$ is called the **innovation** term. We can see that the update equation for new belief is a convex weighted combination of updates from prior and observation, with the Kalman Gain matrix acting as the weight. From the equation for the Kalman Gain matrix, we can see that if observations are noisy ($\sigma_z$ or $R$ is large), then the KG matrix is small and updates rely more on prior. On the other hand if the process is unpredictable (large $\sigma_x$) or prior is unreliable (large $\sigma_t$), the KG matrix is higher and we rely more on the observation. 
+
+
+## Discussion of Where the A, G, and C Matrix Come From 
+
+
+Up to this point, we have discussed inference in the Kalmann filter model; given the 
+model up-front, tell me something about the data. This leaves open where the matrices A, G, and
+C come from, however. This is a similar situation we were in for HMMs: to find the necessary 
+matrices, we must do learning. Using approaches like EM we can interleave learning and inference
+to come across the parameters of interest.
+
+Furthering this comparison to HMMs, the Rauch-Tung-Strievel algorithm allows us to perform "exact off-line inference in an LDS", and is essentially a "Guassian analog of the forwards-backwards" algorithm.
```

Comment:
please add more details here perhaps a citation to a paper

Reply:
Saw this comment in the email I received and addressed it then, but did not see it here earlier.

Please see the comment made above regarding "
Derivation of A, G, and C Matrices"
As stated there, the content there is intentionally high-level to begin with.